### PR TITLE
docs: update link in Moderator Handbook

### PR DESCRIPTION
**Pull request by lasjorg**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

The current staff forum link is wrong.

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
This link is for forum mods only, right? Maybe it'd be beneficial to add information about that by the link.

File: docs/moderator-handbook.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ Forum moderators can delete user's posts. You should only do this for the follow
 
 For the first spam post of a user, send them a message explaining the problem, and remove the link or post as appropriate. Leave a note on the user's profile explaining the action you have taken. If the problem persists, then quietly block the user from posting (using the silence option on the User Admin panel). Send the user a warning with the Code of Conduct. Check the box in the private message indicating that your message is a "formal warning."
 
-You can ask questions and report incidents in the [staff forum section](https://forum.freecodecamp.com/c/staff).
+You can ask questions and report incidents in the [staff forum section](https://forum.freecodecamp.org/c/mod-team/4).
```
Comment (raisedadead) [inline docs/moderator-handbook.md:180]:
```suggestion
As a moderator, you can ask questions and report incidents in the [staff forum section](https://forum.freecodecamp.org/c/mod-team/4).
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
I agree with @gikf - how about something like this.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**